project-codeflare/codeflare-sdk
# Rhoaieng 7141

# Issue link


[RHOAIENG-7141](https://issues.redhat.com/browse/RHOAIENG-7141)
# What changes have been made

`get_cluster` should not fail if there is no default local_queue in `RayCluster`
# Verification steps



Start a new `RayCluster` with a LocalQueue specified and no default one, and run:

`
get_cluster("raytest")
`

Should not got error:
`
Default Local Queue with kueue.x-k8s.io/default-queue: true annotation not found please create a default Local Queue or provide the local_queue name in Cluster Configuration`
## Checks
- Testing Strategy
